client/src/components/atoms/Button.module.scss

```diff
@@ -25,21 +25,21 @@
         color: $brand-white;
         background: $brand-grey-light;
         text-decoration: none;
-        transform: translate3d(0, -.05rem, 0);
-        box-shadow: 0 12px 30px 0 rgba(0, 0, 0, .1);
+        transform: translate3d(0, -0.05rem, 0);
+        box-shadow: 0 12px 30px 0 rgba(0, 0, 0, 0.1);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)